Thunder and Plugin bump ups

HaseenaSainul opened the pull request.

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered-wpewebkit/968/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/19892/ started.

rdkcmf-jenkins commented:
Smoke tests for ['build-rpi-hybrid-nosrc-thunder-next'] added.
https://jenkins.cmf.code.rdkcentral.com/job/test-components-flow/60739/

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered-wpewebkit/972/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/19983/ started.

rdkcmf-jenkins commented:
build-rpi-hybrid-nosrc-thunder-next Console Log uploaded to Gist URL: https://gist.github.com/e2112d6029372501f0e5f2b5a48ce639

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered-wpewebkit/974/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/19994/ started.

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered-wpewebkit/976/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/19995/ started.

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/19996/ started.

rdkcmf-jenkins commented:
Smoke tests for ['build-rpi-hybrid-nosrc-thunder-next'] added.
https://jenkins.cmf.code.rdkcentral.com/job/test-components-flow/60963/

rdkcmf-jenkins commented:
Smoke tests for ['build-rpi-hybrid-nosrc-thunder-next'] added.
https://jenkins.cmf.code.rdkcentral.com/job/test-components-flow/60964/

rdkcmf-jenkins commented:
Smoke tests for ['build-rpi-hybrid-nosrc-thunder-next'] added.
https://jenkins.cmf.code.rdkcentral.com/job/test-components-flow/60965/

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered-wpewebkit/981/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/20002/ started.

rdkcmf-jenkins commented:
'['build-rpi-hybrid-nosrc-thunder-next']' Smoke test FAILURE, investigation pending.
Flash Thunder [SUCCESS]
Thunder - Video Test [FAILURE]
RDKV Thunder Baeble Music App test [FAILURE]
https://jenkins.cmf.code.rdkcentral.com/job/test-raspberrypi-thunder-flow/6244/

rdkcmf-jenkins commented:
'['build-rpi-hybrid-nosrc-thunder-next']' Smoke test FAILURE, investigation pending.
Flash Thunder [SUCCESS]
Thunder - Video Test [FAILURE]
RDKV Thunder Baeble Music App test [FAILURE]
https://jenkins.cmf.code.rdkcentral.com/job/test-raspberrypi-thunder-flow/6245/

rdkcmf-jenkins commented:
'['build-rpi-hybrid-nosrc-thunder-next']' Smoke test FAILURE, investigation pending.
Flash Thunder [SUCCESS]
Thunder - Video Test [FAILURE]
RDKV Thunder Baeble Music App test [FAILURE]
https://jenkins.cmf.code.rdkcentral.com/job/test-raspberrypi-thunder-flow/6246/

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered-wpewebkit/983/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/20005/ started.

rdkcmf-jenkins commented:
Smoke tests for ['build-rpi-hybrid-nosrc-thunder-next'] added.
https://jenkins.cmf.code.rdkcentral.com/job/test-components-flow/60990/

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered-wpewebkit/984/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/20011/ started.

rdkcmf-jenkins commented:
Smoke tests for ['build-rpi-hybrid-nosrc-thunder-next'] added.
https://jenkins.cmf.code.rdkcentral.com/job/test-components-flow/60991/

rdkcmf-jenkins commented:
'['build-rpi-hybrid-nosrc-thunder-next']' Smoke Testing voting skipped due to regressions under investigation . , investigation pending.
Flash Thunder [SUCCESS]
Thunder - Video Test [FAILURE]
RDKV Thunder Baeble Music App test [FAILURE]
https://jenkins.cmf.code.rdkcentral.com/job/test-raspberrypi-thunder-flow/6267/

HaseenaSainul commented:
Hi @rdkcmf-jenkins, in the latest meta-wpe, we used certain plugins from rdk-services and removed these from thundernanoservices. so for that we added build support of these plugins from meta-wpe/recipe-wpe/wpeframework/wpeframework-rdkservices_git.bb. But found that these are masked from meta-cmf-raspberrypi layer. Could you please unmask this ?

rdkcmf-jenkins commented:
Smoke tests for ['build-rpi-hybrid-nosrc-thunder-next'] added.
https://jenkins.cmf.code.rdkcentral.com/job/test-components-flow/61010/

rdkcmf-jenkins commented:
'['build-rpi-hybrid-nosrc-thunder-next']' Smoke Testing voting skipped due to regressions under investigation . , investigation pending.
Flash Thunder [SUCCESS]
Thunder - Video Test [FAILURE]
RDKV Thunder Baeble Music App test [FAILURE]
https://jenkins.cmf.code.rdkcentral.com/job/test-raspberrypi-thunder-flow/6277/

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered-wpewebkit/990/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component-rdk-next/20100/ started.
